flink/flink-scala-parent/pom.xml

```diff
@@ -1008,29 +1014,14 @@
     </profile>
 
     <profile>
-      <id>hive2</id>
+      <id>hive3</id>
       <activation>
         <activeByDefault>true</activeByDefault>
       </activation>
       <properties>
-        <hive.version>2.3.4</hive.version>
-        <hiverunner.version>4.0.0</hiverunner.version>
+        <hive.version>3.1.2</hive.version>
+        <hiverunner.version>6.1.0</hiverunner.version>
       </properties>
     </profile>
-    <profile>
-      <id>hive1</id>
```

Comment:
I think it requires a dedicated JIRA ticket to remove Hive 1.x on Flink interpreter